specification/metadata/creationdate.md

```diff
@@ -0,0 +1,17 @@
+# Creation Date
+
+Date the schema was created.
+
+The CreationDate MUST be present in the metadata. This MUST be of type Date/Time and MUST NOT contain null values. CreationDate MUST conform to [Date/Time Format](#date/timeformat).
```

Comment:
Leave as a SHOULD? In the case of GCP, they have a column for this in their actual data.

Reply:
There is value for practitioners to get this information in a unique way for all providers.
Moreover, it's more efficient for this information to be at metadata level rather than at data level (same info for all rows of the generated dataset).
Practitioners may add this info as a column in their datasets at ingestion time.

Reply:
I would go for a MUST, regardless of whether this is included in the dataset.